examples/instance_with_config_file/variables.tf

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "The project ID to deploy resources into"
+}
+
+variable "subnetwork_project" {
+  description = "The project ID where the desired subnetwork is provisioned"
+}
+
+variable "subnetwork" {
+  description = "The name of the subnetwork to deploy instances into"
+}
+
+variable "instance_name" {
+  description = "The desired name to assign to the deployed instance"
+  default     = "hello-world-container-vm"
+}
+
+variable "image" {
+  description = "The Docker image to deploy to GCE instances"
+  default     = "gcr.io/google-samples/hello-app:1.0"
+}
+
+variable "image_port" {
+  description = "The port the image exposes for HTTP requests"
+  default     = "8080"
+}
+
+variable "restart_policy" {
```

Comment:
FYI in general we shouldn't really need so many variables for examples - often they should just be hardcoded. I know many of the existing examples overuse variables, but the variables should really only be for things which actually need to change like project ID.

Reply:
I just copied it from instance_with_attached_disk. Should I remove it here or in both places?
Unlike instance_with_attached_disk, I set a default here so it doesn't actually need to be defined.

Reply:
Just because you started with an example doesn't mean you need to copy it exactly - try to make it better as you add new ones at least.